npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker.rb

```diff
@@ -178,7 +172,19 @@ def conflicting_updated_dependencies
           )
         end
 
-        updated_deps
+        # Target dependency should be first in the result to support rebases
+        updated_deps.select { |dep| dep.name == dependency.name } +
+          updated_deps.reject { |dep| dep.name == dependency.name }
+      end
+
+      def top_level_dependency_lookup
```

Comment:
Thought: this seems kind of expensive, maybe we should memoize it? But we can always cross that bridge if we use this method elsewhere.

Reply:
Yup, I figured we'd wait to see if it gets used elsewhere first.